main/guides/getting-started/README.md

```diff
@@ -27,7 +28,7 @@ If `agoric` is not found, then you may need to add the parent directory of where
 This presentation is a good overview of the Agoric SDK setup process,
 though a few details are out of date:
 
-- node version: 12.x is too old; use 16.19.1 or higher
+- node version: 12.x is too old; use 18.16.0 or higher
```

Comment:
Again, let's move any changes to the getting-started section, such as changing the node version, to its own PR.

Reply:
Done.